doc/en/user/source/community/json-ld/output.rst

```diff
@@ -0,0 +1,81 @@
+JSON-LD Output Format
+============================
+
+JSON-LD Output Format
+---------------------
+
+
+In order to obtain a json-ld output format, request's media type must be specified as "application/ld+json".
+From Geoserver UI it is possible to have a preview of the format by choosing JSON-LD format by the layer preview page.
+
+.. figure:: images/json-ld_preview.png
+
+   Dropdown menu in Layer preview page.
+
+
+
+If wfs3 extension is enabled the output format is available from wfs3 collection page as well.
```

Comment:
Please refer to the OGC Features API extension instead, the WFS3 module is dead (we should probably just remove it, but it still has a couple of extra functionalities not yet ported over to the ogcapi-features module, once we move them the module will be removed).

Reply:
fixed